src/main/java/com/thevoxelbox/voxelsniper/brush/type/GenerateTreeBrush.java

```diff
@@ -95,8 +95,8 @@ public void handleCommand(String[] parameters, Snipe snipe) {
         if (firstParameter.equalsIgnoreCase("info")) {
             messenger.sendMessage(ChatColor.GOLD + "This brush takes the following parameters:");
             messenger.sendMessage(ChatColor.AQUA + "/b gt default -- Restores default params.");
-            messenger.sendMessage(ChatColor.AQUA + "/b gt lt [t] -- Sets leaf type to t. (e.g. oak)");
-            messenger.sendMessage(ChatColor.AQUA + "/b gt wt [t] -- Sets wood type to t. (e.g. oak)");
+            messenger.sendMessage(ChatColor.AQUA + "/b gt lt [t] -- Sets leaf type to t. (e.g. oak_leaves)");
+            messenger.sendMessage(ChatColor.AQUA + "/b gt wt [t] -- Sets wood type to t. (e.g. oak_leaves)");
```

Comment:
Wood type is still oak and not oak_leaves ;)

Reply:
That'd be `oak_log` then.